packages/web-components/src/card/index.ts

```diff
@@ -1,7 +1,7 @@
-import { customElement } from '@microsoft/fast-element';
 import { ColorRGBA64, parseColorHexRGB } from '@microsoft/fast-colors';
-import { designSystemProperty, DesignSystemProvider, CardTemplate as template } from '@microsoft/fast-foundation';
+import { designSystemProperty, designSystemProvider, CardTemplate as template } from '@microsoft/fast-foundation';
 import { createColorPalette, DesignSystem } from '@microsoft/fast-components-styles-msft';
+import { FluentDesignSystemProvider } from '..';
```

Comment:
I think we'll want to point to the index file here for safety, sometimes this can create TS issues

Reply:
```suggestion
import { FluentDesignSystemProvider } from '../index';
```